File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    
	    Scanner in = new Scanner(System.in);
	    
	    int hour, min, x;
	    
	    hour = in.nextInt();
	    min = in.nextInt();
	    x = in.nextInt();
	    
	   if (min + x < 60)
	        System.out.println(hour + " " + (min+x));
	   else if (min+x >=60) {
	       int i = (min+x)/60;
	       int l = (min+x)%60;
	       
	       if (hour+i >= 24) {
	       System.out.println(((hour+i)%24) + " " + l);
	       
	       } else {
	           System.out.println((hour+i)+" "+ l);
	       }
	   }

    }
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include <iostream>

using namespace std;

int main()
{
    int a, b, c;
    
    cin >> a >> b >> c;
    


        int total_minute = a*60+b+c;
        int hour = total_minute/60;
        int min = total_minute%60;
        if (hour > 23) {
            cout << hour%24 << " " << min;
        } else {
            cout << hour << " " << min;
        
        }
    



    return 0;
}